### PR TITLE
adding oesign signing engine test

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: tests/myst/signing-engine-test/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1
```
Comment (CyanDevs) [inline tests/myst/signing-engine-test/.gitignore:1]: These numbers appear unrelated?

Comment (salsal97) [reply tests/myst/signing-engine-test/.gitignore:1]: They will be files created by the test, added them here to avoid people pushing them in
